cmd/provider/main.go

```diff
@@ -76,6 +78,13 @@ func main() {
 		Features:                &feature.Flags{},
 	}
 
-	kingpin.FatalIfError(ansible.Setup(mgr, o, *ansibleCollectionsPath, *ansibleRolesPath, *timeout), "Cannot setup Ansible controllers")
+	ao := ansiblerun.SetupOptions{
```

Comment:
The list of parameters that apply only to the ansiblerun controller and not to the config controller was getting too long as I added another flag, so I decided to refactor a bit and out this into a struct